scripts/findReactIntlViolators.js

```diff
@@ -9,7 +9,7 @@ const features = 'formatRelativeTime|FormattedRelativeTime|createIntl|useIntl';
 const allowList = [
     'src/components/time/ReadableTime.js',
     'src/features/presence/PresenceAvatarTooltipContent.tsx',
-    'src/features/presence/PresenceCollaboratorsList.js',
+    'src/features/presence/PresenceCollaborator.js',
```

Comment:
I think this change will require an approval from @box/foundation.

Reply:
@alexkrolick Would you be able to take a look at this change? Thank you!